packages/lib/link/messages.js

```diff
@@ -1150,6 +1152,7 @@ messages.instanceStatusChanged = new Event({
 		"status": { enum: [
 			"stopped", "starting", "running", "stopping", "creating_save", "exporting_data",
 		]},
+		"game_port": { type: ["null", "integer"] },
```

Comment:
Properties added to `eventProperties` are by default required, which will cause slaves connecting from the previous version to fail. You need to add an `eventRequired` array with a list of the previous properties and handle game_port missing in the event handler.